ci/cli_build_release.sh

```diff
@@ -86,44 +85,23 @@ if [[ ! "$release_tag" == *"-"* ]]; then
    # TODO(james): Add push to docker hub/quay.io.
 fi
 
-gpg --no-tty --batch --yes --import "${BUILDBOT_GPG_KEY_FILE}"
-
-write_artifacts_to_gcs() {
-  output_path=$1
-  copy_artifact_to_gcs "${output_path}" "${darwin_amd64_binary}" "cli_darwin_amd64_unsigned"
-  copy_artifact_to_gcs "${output_path}" "${darwin_arm64_binary}" "cli_darwin_arm64_unsigned"
-  copy_artifact_to_gcs "${output_path}" "${linux_binary}" "cli_linux_amd64"
+upload_artifacts() {
+  version="$1"
+  export GCS_ARTIFACT_OPTS="-h 'Content-Disposition:filename=px'"
```

Comment:
We always had this. See the deleted `gcs_utils.sh` file. I think this means if you download the link, the downloaded file will be called `px`, regardless of if its actually called `cli_darwin_amd64` or `cli_linux_amd64`. I'm happy to remove it, since this won't work on the github releases artifacts anyways.

Reply:
After discussion, will remove.